app/src/main/java/com/cornellappdev/scoop/ui/onboarding/OnboardingView2.kt

```diff
@@ -110,7 +109,7 @@ fun OnboardingView2(pagerState: PagerState) {
                 verticalAlignment = Alignment.Bottom,
                 horizontalArrangement = Arrangement.Center,
             ) {
-                OnboardingFooter(carIndex = pagerState.currentPage)
+                MovingCarFooter(carIndex = pagerState.currentPage, 6)
             }
         }
```

Comment:
These `OnboardingViewX` composables are weird. Maybe rename them or add a spec saying what they are, cuz their names are definitely not self documenting.